config/config.go

```diff
@@ -55,11 +55,18 @@ var (
 )
 
 // Get returns config.
-func Get() *Config {
+func Get(env string) *Config {
```

Comment:
```suggestion
func Get(env ...string) *Config {
```

config.Get("") виглядає неінтуітивно, пропоную config.Get() як альтернативу

Reply:
Згоден, дякую🙌